runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TranslationUtils.java

```diff
@@ -192,22 +192,30 @@ public Boolean call(Tuple2<TupleTag<V>, WindowedValue<?>> input) {
    * @return a map of tagged {@link BroadcastHelper}s and their {@link WindowingStrategy}.
    */
   public static Map<TupleTag<?>, KV<WindowingStrategy<?, ?>, BroadcastHelper<?>>>
-      getSideInputs(List<PCollectionView<?>> views, EvaluationContext context) {
+  getSideInputs(List<PCollectionView<?>> views, EvaluationContext context) {
```

Comment:
It appears that this can be package-private, which is also nicer since it has an overriding implementation.